pkg/parser/addr_parser.go

```diff
@@ -5,9 +5,9 @@ import (
 	"strings"
 )
 
-// ParseAddr -
-// @todo - this is fairly crude, could be improved.
+// ParseAddr parse a string into a Signature
 func ParseAddr(signature string) (*types.Signature, error) {
+	// @todo - this is fairly crude, could be improved.
```

Comment:
Agreed! Not pretty 😂

Reply:
Oh, I just moved that line from outside the function to the inside so it didnt' show up in godoc.  It was already in the code, lol.  Not may addition!